### PR TITLE
More and more checkstyle

### PR DESCRIPTION
**Pull request by loosebazooka**
- SummaryJavadoc

Part of #2272 

notable: single line `@returns ...` -> `Returns ...` becauseof  [google code style](https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment)

### CODE REVIEW

File: jib-core/src/main/java/com/google/cloud/tools/jib/http/Response.java [hunk 1]
```diff
@@ -33,12 +33,14 @@
...
   }
 
-  /** @return the HTTP status code of the response */
+  /** Returns the HTTP status code of the response. */
   public int getStatusCode() {
     return httpResponse.getStatusCode();
   }
 
   /**
+   * Returns an list of the header string values for the given header name.
```
Comment (chanseokoh) [inline jib-core/src/main/java/com/google/cloud/tools/jib/http/Response.java]:
```suggestion
   * Returns a list of the header string values for the given header name.
```

File: jib-core/src/main/java/com/google/cloud/tools/jib/registry/RegistryEndpointRequestProperties.java [hunk 2]
```diff
@@ -26,6 +26,8 @@
   @Nullable private final String sourceImageName;
 
   /**
+   * New properties with no sourceImageName.
```
Comment (chanseokoh) [inline jib-core/src/main/java/com/google/cloud/tools/jib/registry/RegistryEndpointRequestProperties.java]:
```suggestion
   * New properties.
```

File: jib-core/src/main/java/com/google/cloud/tools/jib/registry/RegistryErrorExceptionBuilder.java [hunk 3]
```diff
@@ -62,7 +62,7 @@ private static String getReason(@Nullable String errorCodeString, @Nullable Stri
     }
   }
 
-  /** @param method the registry method that errored */
+  /** Create a new builder with information about the method that errored. */
```
Comment (chanseokoh) [inline jib-core/src/main/java/com/google/cloud/tools/jib/registry/RegistryErrorExceptionBuilder.java]:
```suggestion
  /** Creates a new builder with information about the method that errored. */
```

File: jib-core/src/main/java/com/google/cloud/tools/jib/registry/RegistryErrorExceptionBuilder.java [hunk 4]
```diff
@@ -71,7 +71,7 @@ private static String getReason(@Nullable String errorCodeString, @Nullable Stri
     errorMessageBuilder.append(" but failed because: ");
   }
 
-  /** @param method the registry method that errored */
+  /** Create a new builder with information about the method that errored. */
```
Comment (chanseokoh) [inline jib-core/src/main/java/com/google/cloud/tools/jib/registry/RegistryErrorExceptionBuilder.java]:
```suggestion
  /** Creates a new builder with information about the method that errored. */
```

File: jib-core/src/main/java/com/google/cloud/tools/jib/registry/credentials/DockerConfig.java [hunk 5]
```diff
@@ -116,7 +116,9 @@ DockerCredentialHelper getCredentialHelperFor(String registry) {
   }
 
   /**
-   * Registry alias matches in the following order:
+   * Gets alias matches for a registry.
```
Comment (chanseokoh) [inline jib-core/src/main/java/com/google/cloud/tools/jib/registry/credentials/DockerConfig.java]: These are not aliases.
```suggestion
   * Gets registry matchers for a registry.
```

File: jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/TestProject.java [hunk 6]
```diff
@@ -46,7 +46,7 @@ public TestProject(String projectDir) {
     this.projectDir = projectDir;
   }
 
-  /** Get the project root resolved as a real path */
+  /** Get the project root resolved as a real path. */
```
Comment (chanseokoh) [inline jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/TestProject.java]:
```suggestion
  /** Gets the project root resolved as a real path. */
```

File: jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/HelpfulSuggestions.java [hunk 7]
```diff
@@ -22,6 +22,8 @@
 public class HelpfulSuggestions {
 
   /**
+   * Generate message for when "target image" isn't configured.
```
Comment (chanseokoh) [inline jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/HelpfulSuggestions.java]:
```suggestion
   * Generates message for when "target image" isn't configured.
```

File: jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/MainClassResolver.java [hunk 8]
```diff
@@ -78,6 +78,8 @@ public static String resolveMainClass(
   }
 
   /**
+   * Check if a string is a valid Java class name.
```
Comment (chanseokoh) [inline jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/MainClassResolver.java]:
```suggestion
   * Checks if a string is a valid Java class name.
```

File: jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/PluginConfigurationProcessor.java [hunk 9]
```diff
@@ -363,7 +363,9 @@ static JavaContainerBuilder getJavaContainerBuilderWithBaseImage(
   }
 
   /**
-   * Compute the container entrypoint, in this order:
+   * Compute the container entrypoint.
```
Comment (chanseokoh) [inline jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/PluginConfigurationProcessor.java]:
```suggestion
   * Computes the container entrypoint.
```

File: jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/VersionChecker.java [hunk 10]
```diff
@@ -129,6 +129,8 @@ public boolean compatibleVersion(String acceptableVersionRange, String actualVer
   }
 
   /**
+   * Parse and return a version object.
```
Comment (chanseokoh) [inline jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/VersionChecker.java]:
```suggestion
   * Parses and returns a version object.
```

**Comment by chanseokoh** [comment]
There are Javadoc errors.
```
/home/travis/build/GoogleContainerTools/jib/jib-core/src/main/java/com/google/cloud/tools/jib/http/Response.java:37: warning: no @return
```

**Comment by loosebazooka** [comment]
Interesting, the javadoc error conflicts with google's official style guide recommendation. :(

**Comment by chanseokoh** [comment]
So I guess `@return` is optional in Google style?

**Comment by loosebazooka** [comment]
I think google style, checkstyle and javadoc are in conflict here, I'll just fix it to satisfy all 3 (even if redundant).

**Pull request merged**